src/syscheckd/src/db/src/fimDBSpecialization.h

```diff
@@ -225,8 +225,8 @@ class FIMDBCreator<OSType::WINDOWS> final
 {
     public:
         static void setLimits(std::shared_ptr<DBSync> DBSyncHandler,
-                              const unsigned int& fileLimit,
-                              const unsigned int& registryLimit)
+                              const int fileLimit,
```

Comment:
fileLimit and registryLimit check > 0 should also consider >= 0 condition, am i right?

Reply:
good catch, those conditions should be removed, this is handled with exceptions in dbsync.